lib/block.ml

```diff
@@ -158,15 +158,20 @@ let pp_lines syntax t =
   in
   Fmt.(list ~sep:(any "\n") pp)
 
-let lstrip string =
-  let hpad = Misc.hpad_of_lines [ string ] in
-  Astring.String.with_index_range string ~first:hpad
+let lstrip strings =
+  let hpad = Misc.hpad_of_lines strings in
+  List.map
+    (fun string ->
+      let min a b = if a < b then a else b in
```

Comment:
Given this was added in OCaml 4.13, I would suggest adding it to `Util`, as `Util.Int.min` with the same signature as the Stdlib, so if we end up dropping compatibility with older versions we can easily remove it.